proposals/0303-swiftpm-extensible-build-tools.md

```diff
@@ -3,51 +3,55 @@
 * Proposal: [SE-0303](0303-swiftpm-extensible-build-tools.md)
 * Authors: [Anders Bertelrud](https://github.com/abertelrud), [Konrad 'ktoso' Malawski](https://github.com/ktoso), [Tom Doron](https://github.com/tomerd)
 * Review Manager: [Tom Doron](https://github.com/tomerd)
-* Status: **Implemented (Swift 5.6)**
-* Decision Notes: [Acceptance](https://forums.swift.org/t/accepted-se-0303-package-manager-extensible-build-tools/)
-* Pitch and Discussion: [Pitch: SwiftPM Extensible Build Tools](https://forums.swift.org/t/pitch-swiftpm-extensible-build-tools/44715)
-* Review: [1](https://forums.swift.org/t/se-0303-package-manager-extensible-build-tools/) [2](https://forums.swift.org/t/se-0303-2nd-review-package-manager-extensible-build-tools/)
-* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/878e496eb799fa407ad704d89fb401952fe8fd02/proposals/0303-swiftpm-extensible-build-tools.md)
+* Status: **Revision Awaiting Review**
+* Revision Pitch and Discussion: [Pitch: Amend SE-0303 Plugin API to Use `@main` for Plugin Entry Point](https://forums.swift.org/t/pitch-amend-se-0303-plugin-api-to-use-main-for-plugin-entry-point/51250)
+* Original Pitch and Discussion: [Pitch: SwiftPM Extensible Build Tools](https://forums.swift.org/t/pitch-swiftpm-extensible-build-tools/44715)
+* Previous Reviews: [1](https://forums.swift.org/t/se-0303-package-manager-extensible-build-tools/), [2](https://forums.swift.org/t/se-0303-2nd-review-package-manager-extensible-build-tools/)
+* Previous Revisions:, [1](https://github.com/apple/swift-evolution/blob/878e496eb799fa407ad704d89fb401952fe8fd02/proposals/0303-swiftpm-extensible-build-tools.md) [2](https://github.com/apple/swift-evolution/blob/38731efc140a53553aff923a6616a1dee28c973a/proposals/0303-swiftpm-extensible-build-tools.md)
```

Comment:
* Original Pitch  and Discussion: [Pitch: SwiftPM Extensible Build Tools](https://forums.swift.org/t/pitch-swiftpm-extensible-build-tools/44715)
 * Original Reviews: 
    * [First review](https://forums.swift.org/t/se-0303-package-manager-extensible-build-tools/)
    * [Second review](https://forums.swift.org/t/se-0303-2nd-review-package-manager-extensible-build-tools/)
* Amendment Pitch and Discussion: [Pitch: Amend SE-0303 Plugin API to Use `@main` for Plugin Entry Point](https://forums.swift.org/t/pitch-amend-se-0303-plugin-api-to-use-main-for-plugin-entry-point/51250)
* Previous Revisions:
   *  [First revision](https://github.com/apple/swift-evolution/blob/878e496eb799fa407ad704d89fb401952fe8fd02/proposals/0303-swiftpm-extensible-build-tools.md) 
   *  [Second revision](https://github.com/apple/swift-evolution/blob/38731efc140a53553aff923a6616a1dee28c973a/proposals/0303-swiftpm-extensible-build-tools.md)